docs/try-flink/table_api.md

```diff
@@ -44,6 +44,13 @@ It also assumes that you are familiar with basic relational concepts such as `SE
 If you get stuck, check out the [community support resources](https://flink.apache.org/community.html).
 In particular, Apache Flink's [user mailing list](https://flink.apache.org/community.html#mailing-lists) consistently ranks as one of the most active of any Apache project and a great way to get help quickly. 
 
+
+### Docker on windows
```

Comment:
We capitalize all proper words in titles along with a new line between the header and the first 

```suggestion
### Docker on Windows

```